data/raw/data_prediction.sql

```diff
@@ -54,109 +56,25 @@ FROM [PUB].[no_show].[HealthcareService] SUBAGENDA JOIN [PUB].[no_show].[Healthc
 WHERE 1=1
     AND SUBAGENDA.active = 1
     AND SUBAGENDA.identifier_system = 'https://metadata.umcutrecht.nl/ids/HixSubAgenda'
-    AND SUBAGENDA.identifier_value NOT IN (
-        'Z10351', 'Z10330', 'Z10307', 'Z10362', 'Z10438',  -- Hartgroepen 1 t/m 5 (REV)
-        'ZH0082', 'ZH0085', 'ZH0086', 'ZH0087', 'ZH0088',  -- Hartgroepen 1 t/m 5 (FYS)
-        'Z10455',  -- Behandelaar CMH
-        -- Longziekten
-        'ZH0302', 'Z01613', 'Z01577',  -- LAB Longziekten
-        'Z01564', 'Z01565', 'Z01566', 'Z01582', 'Z01583', 'Z01584', 'Z01585', 'Z01622', 'Z01623', 'Z01624', 'Z01625', 'Z01571', 'Z01570', -- Dagbehandeling
-        -- Huisbezoeken van Centrum voor Thuisbeademing
-        'Z04789', 'Z04792', 'Z04790', 'Z04791', 'Z04764', 'Z04765', 'Z04760', 
-        'Z04766', 'Z04757', '029756', 'Z04745', 'Z04763', 'ZH0633', 'Z04761',
-        'Z04787', 'Z04747', 'Z04767', 'Z04746', 'Z04769', 'Z04758', 'Z04788',
-        'Z04786', 'Z04759',
-        -- Poli blauw
-        'Z06676', -- Ciliopathie
-        'Z07053', -- PMC
-        'Z07081', -- Research DER
-        'Z07078', -- Bioday kind
-        'Z04778', -- afgifteloket van het lab.
-        'Z04755', -- afgifteloket van het laboratorium
-        -- Cardiologie
-        'Z01282', -- OACAR
-        'Z01254', -- RPM
-        'Z01724', -- CTC,
-        'Z01707', -- post CTC PA,
-        'Z01704', -- POST CTC Chirurg,
-        'Z01270', -- HTX jaarkaart na,
-        'Z01269', -- HTZ jaarkaart pre,
-        'Z01278', -- HTX post-operatief,
-        'Z01180', -- HTXV, 
-        'Z01222', -- HTXVS,
-        'Z01208', -- LVADVS
-        -- HTX subagenda's
-        'Z01178', '029708', '029710', '029712', '029713', '029711', '029707', 'Z01226',
-        --- GUCH subagenda's
-        'ZH0192', 'ZH0561', 'Z01316', 'Z01188', 'Z01234'
-    )
     AND HOOFDAGENDA.identifier_system = 'https://metadata.umcutrecht.nl/ids/HixAgenda'
     AND HOOFDAGENDA.active = 1
-    AND (
-        HOOFDAGENDA.identifier_value IN (
+    AND HOOFDAGENDA.identifier_value IN (
+            -- Revalidatie en sport
+            'ZH0307',  -- RF&S Revalidatiegeneeskunde
+            'ZH0435',  -- RF&S Sportgeneeskunde
```

Comment:
We'll probably remove this when every poli is included right? Currently we still need to update both the config and query if we add more poli's

Reply:
Yes, but for now I want to reduce the size of the data with only this filter